client/contracts/bindings/consensus/consensus.go

```diff
@@ -77,7 +62,12 @@ type mUSD struct {
 }
 
 func NewMUSD(contractBackend bind.ContractBackend, chainID *big.Int) (*mUSD, error) {
-	mtoken, err := NewMiningToken(mapMiningTokenToAddr[chainID.Uint64()], contractBackend)
+	addr, err := getAddressFromKNS("miningtoken.kowala", contractBackend)
```

Comment:
add "miningtoken.kowala" to the params package (ex: system_domains.go/domains.go)